apps/utils/scripts/run-bfd-server

```diff
@@ -85,11 +85,14 @@ debug_enabled=false
 debug_suspend=n
 db_host=localhost
 db_port=5432
+username=
+password=
```

Comment:
I think we can default these to the local ones just like the db_host has a default here
```suggestion
username=bfd
password=InsecureLocalDev
```